app/src/main/java/com/team3663/scouting_app/activities/Match.java

```diff
@@ -642,7 +628,7 @@ private void initNotMoving() {
             } else {
                 Globals.EventLogger.LogEvent(Constants.Events.ID_NOT_MOVING_END, 0,0,false);
                 matchBinding.switchNotMoving.setBackgroundColor(Constants.Match.BUTTON_COLOR_NORMAL);
-                Achievements.data_IdleTime += System.currentTimeMillis() - starttime_not_moving;
+                Achievements.data_IdleTime += (int)(System.currentTimeMillis() - starttime_not_moving);
```

Comment:
there was a compiler WARNING that we're implicitly converting from long to int.  By making an explicit casting, the warning went away.  I acknowledge the long->int lossy-ness but since we're subtracting two longs, it'll fit into an int.  :)